### DISCUSSION TITLE
Default models diversity

### DISCUSSION THREAD

**Discussion by TuomasJaa** [lllyasviel/Fooocus#811] [Q&A]
I have just starting to experiment with this system. But alredy I have noticed that default models prefer to generate white men with beards. In group photos even when you ask for men with no beard all you get is men with beards. And most of a time you have specify that you want also women on the picture. If you ask more divercity, all you get is white bearded men.
Is there anyway to get more diversity?

**Comment by lllyasviel** [comment]
hi
SD does not recognize "no" like dalle, or "--no" like MJ. if you type "with no beard", then the model will see the keyword "beard" and generate anyways
unwanted prompts needs to be in "Negative prompts"

also it seems that `young` can solve this
